### PR TITLE
games-fps/eduke32: bump to version 20181010.7076

### PR DESCRIPTION
**Pull request by ConiKost**
Closes: https://bugs.gentoo.org/668606
Signed-off-by: Conrad Kostecki <conrad@kostecki.com>
Package-Manager: Portage-2.3.50, Repoman-2.3.11

### CODE REVIEW

File: games-fps/eduke32/eduke32-20181010.7067.ebuild [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="6"
+
+inherit desktop eapi7-ver gnome2-utils toolchain-funcs
+
+MY_BUILD="$(ver_cut 2)"
+MY_DATE="$(ver_cut 1)"
+
+MY_PN_HRP="duke3d_hrp"
```
Comment (mgorny) [inline games-fps/eduke32/eduke32-20181010.7067.ebuild]: What's the purpose of all those variables?

File: games-fps/eduke32/eduke32-20181010.7067.ebuild [hunk 2]
```diff
@@ -0,0 +1,247 @@
...
+
+src_install() {
+	local binary
+	local binaries=(
+		eduke32
+		mapster32
+		"${FILESDIR}"/eduke32-bin
+	)
+	for binary in "${binaries[@]}"; do
+		dobin "${binary}"
```
Comment (mgorny) [inline games-fps/eduke32/eduke32-20181010.7067.ebuild]: `dobin` accepts multiple parameters, so you don't need a loop.

File: games-fps/eduke32/eduke32-20181010.7067.ebuild [hunk 3]
```diff
@@ -0,0 +1,247 @@
...
+			map2stl
+			md2tool
+			mkpalette
+			transpal
+			unpackssi
+			wad2art
+			wad2map
+		)
+		for tool in "${tools[@]}"; do
+			dobin ${tool}
```
Comment (mgorny) [inline games-fps/eduke32/eduke32-20181010.7067.ebuild]: Likewise.

File: games-fps/eduke32/eduke32-20181010.7067.ebuild [hunk 4]
```diff
@@ -0,0 +1,247 @@
...
+
+	local DOCS=( package/sdk/samples/*.txt source/build/doc/*.txt source/duke3d/src/lunatic/doc/*.txt )
+	if use hrp; then
+		DOCS+=( "${WORKDIR}"/hrp_readme.txt "${WORKDIR}"/hrp_todo.txt )
+	fi
+	if use offensive; then
+		DOCS+=( "${WORKDIR}"/xxx_readme.txt )
+	fi
+	if use opl; then
+		DOCS+=( "${WORKDIR}"/readme.txt )
```
Comment (mgorny) [inline games-fps/eduke32/eduke32-20181010.7067.ebuild]: Isn't this a bad name for subcomponent readme?

File: games-fps/eduke32/eduke32-20181010.7067.ebuild [hunk 5]
```diff
@@ -0,0 +1,247 @@
...
+		DOCS+=( "${WORKDIR}"/readme/music_readme.txt )
+	fi
+	if use voxels; then
+		DOCS+=( "${WORKDIR}"/voxelpack_readme.txt )
+	fi
+	einstalldocs
+}
+
+pkg_preinst() {
+	gnome2_icon_savelist
```
Comment (mgorny) [inline games-fps/eduke32/eduke32-20181010.7067.ebuild]: This is no longer used.

Comment (ConiKost) [reply games-fps/eduke32/eduke32-20181010.7067.ebuild]: Thanks, removed.

File: games-fps/eduke32/eduke32-20181010.7067.ebuild (hunk 4 above)
Comment (ConiKost) [reply games-fps/eduke32/eduke32-20181010.7067.ebuild]: Yes, you are right. I've renamed it now to `opl_readme.txt`, to make it more consistent.

File: games-fps/eduke32/eduke32-20181010.7067.ebuild (hunk 3 above)
Comment (ConiKost) [reply games-fps/eduke32/eduke32-20181010.7067.ebuild]: Done.

File: games-fps/eduke32/eduke32-20181010.7067.ebuild (hunk 2 above)
Comment (ConiKost) [reply games-fps/eduke32/eduke32-20181010.7067.ebuild]: Done.

File: games-fps/eduke32/eduke32-20181010.7067.ebuild (hunk 1 above)
Comment (ConiKost) [reply games-fps/eduke32/eduke32-20181010.7067.ebuild]: These are addon packages for eduke32. All are used in `SRC_URI`.
Yes, I could write MY_PN_* and MY_PV_* directly into `SRC_URI`, but I think, this makes it better to read eBuild this way.

Comment (mgorny) [reply games-fps/eduke32/eduke32-20181010.7067.ebuild]: MY_PV might make sense but MY_PN is pretty much writing 'foo' as '${FOO}'.

Comment (ConiKost) [reply games-fps/eduke32/eduke32-20181010.7067.ebuild]: Ok, removed MY_PN, but left MY_PV.

**Comment by ConiKost** [comment]
No, `make test` does not run any tests. Instead, it tries to build some sort of a test engine/game. It's unsupported by upstream, so disabled it.

**Pull request closed without merging**